show-links.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Go through packages and for each enumrate the node_modules that are symlinked
+
+TITLE_COLOR='\033[0;36m'
+NO_COLOR='\033[0m'
+
+PACKAGES="broker
+cli-tools
+client
+network
+protocol
+test-utils
+"
+for p in $PACKAGES
+do
```

Comment:
Oh actually, you probably want `lerna exec`, something like this:

`npx lerna exec --no-bail 'find node_modules -maxdepth 1 -type l -exec basename {} \;'`

Reply:
Running that command gets me the entire list of symlinked packages but without the associated "title", i.e.:

```
streamr-test-utils
streamr-client-protocol
streamr-test-utils
streamr-test-utils
streamr-network
streamr-test-utils
streamr-client-protocol
streamr-client
streamr-client
```

So not sure how I would use this, if you got some idea, feel free to edit.